ops/services/alerts/app_service_metrics/function_triggers.tf

```diff
@@ -0,0 +1,92 @@
+resource "azurerm_monitor_alert_processing_rule_suppression" "function_triggers_suppression" {
+  name                = "${var.env}-function-triggers-suppression"
+  resource_group_name = var.rg_name
+  scopes              = [data.azurerm_resource_group.app.id]
+
+  condition {
+    target_resource_type {
+      operator = "Equals"
+      values   = ["Microsoft.Web/azureFunctions"]
+    }
+    severity {
+      operator = "Equals"
+      values   = ["Sev4"]
+    }
+  }
+
+  schedule {
+    effective_from  = "2022-01-01T01:02:03"
+    effective_until = "2022-02-02T01:02:03"
```

Comment:
Should these dates be in 2022? Do we even need to have a `effective_from` / `effective_within` for this rule suppression?

Reply:
Technically, this portion is optional based on the current version of the provider, but successful testing was done with it in place. Removing it will require another 24-hour period to perform live testing. 

The date range is just fine as-is. All this does in code is specify a period during which suppression will be in effect, such as a manually-specified maintenance window. It has no effect since it is in the past, but can be leveraged again for us to use in the future if we need to disable this alert for a set period of time. I would personally chose to leave it in place for that reason. (It saves us from having to look it up later, again!)

The "recurrence" block sets the automatic schedule for the suppression outside of a manually-specified maintenance window. These values all stack, rather than taking away from each other.